.github/workflows/build-tarball.yml

```
@@ -72,10 +72,10 @@ jobs:
         run: |
           git config --global core.autocrlf true
       - uses: actions/checkout@v2
-      - name: Set up Python 2.7
+      - name: Set up Python 3.8
         uses: actions/setup-python@v1
```

```suggestion
        uses: actions/setup-python@v2
```
https://github.com/actions/setup-python/releases